subsys/logging/Kconfig

```diff
@@ -279,6 +280,58 @@ config LOG_FRONTEND
 	  When enabled, logs are redirected to a custom frontend instead
 	  of being processed by the logger.
 
+config LOG_BACKEND_FS
+	bool "Enable file system backend"
+	depends on FILE_SYSTEM
```

Comment:
Has this PR been tested with anything else than littlefs ?

Reply:
should probably be, based on the description of this PR https://github.com/zephyrproject-rtos/zephyr/pull/27486#issue-465992198.
> This PR introduces a new backend based on littleFS.

```suggestion
	depends on FILE_SYSTEM_LITTLEFS
```

Reply:
Replaced.